docs/source/troubleshooting/troubleshooting-misc.adoc

```diff
@@ -59,4 +59,37 @@ For example:
 
 ----
 minishift start --iso-url centos
-----
\ No newline at end of file
+----
+
+[[x509-certificate-is-valid-for-10.0.2.15-127.0.0.1-192.168.99.100-not-192.168.99.101]]
+== x509 certificate is valid for 10.0.2.15, 127.0.0.1, 172.17.0.1, 172.30.0.1, 192.168.99.100, not 192.168.99.101
+
+Starting a stopped {project} VM can produce the following X.509 certificate error:
+
+----
+$ minishift start
+-- Checking if requested hypervisor 'kvm' is supported on this platform ... OK
+-- Checking the ISO URL ... OK
+-- Starting local OpenShift cluster using 'kvm' hypervisor ...
+-- Starting Minishift VM ......................... OK
+[...]
+FAIL
+   Error: cannot access master readiness URL https://192.168.99.101:8443/healthz/ready
+   Details:
+     No log available from "origin" container
+
+   Caused By:
+     Error: Get https://192.168.99.101:8443/healthz/ready: x509: certificate is valid for 10.0.2.15, 127.0.0.1, 172.17.0.1, 172.30.0.1, 192.168.99.100, not 192.168.99.101
+Error during 'cluster up' execution: Error starting the cluster.
+----
+
+The reason for the above error is that OpenShift cluster certificates contain the IP of the {project} VM.
+The certificates are generated only when {project} VM is freshly started.
+So when a restart changes the {project} VM IP the certificates are not valid anymore.
```

Comment:
"The certificates become invalid when a restart changes the IP of the {project} VM."

Reply:
Maybe the cause might make this clearer: The lease on the IP address expires, and therefore the VM's IP address will change. It is not the restart itself that changes the address... and it does not always happen.

TL;dr => "Upon restart the {project} VM might get assigned a new IP address. If this happens the certificate is not valid anymore".